jobs/workflow_test_release.groovy

```diff
@@ -1,4 +1,6 @@
-evaluate(new File("${WORKSPACE}/common.groovy"))
+def workspace = new File(".").getAbsolutePath()
+if (!new File("${workspace}/common.groovy").exists()) { workspace = "${WORKSPACE}"}
```

Comment:
do we want to go a little further and check if the file is readable, or is `.exists()` sufficient enough for our use case?